assets/mod_info.json

```diff
@@ -3,7 +3,7 @@
     "name": "Stellar Networks",
     "author": "Jaghaimo",
     "utility": true,
-    "version": "3.2.1",
+    "version": "3.2.1-hf119",
```

Comment:
N2S update before merge.

Reply:
Personally I'd either merge all of those bugfixes to a release branch to test it in bulk before merging to master and releasing as 3.2.2. for now just merge all to master and bump version?

Reply:
I responded to other message, I will merge those and make 3.2.2 - FWIW I was testing all those changes locally, but noted, will make release branches in the future.